application.yml

```diff
@@ -0,0 +1,6 @@
+com:
+  kb:
+    jwt:
+      aws:
+        userPoolId: "us-east-2_ijXLzbX6J"
```

Comment:
Configure, or remove

Reply:
The simple command **System.getenv("variable-name")** can read an environment variable.
Hence should I create/use an 'application.properties' or similar to inform the DevOps that this variable should be created/updated before the launch ?

Reply:
`application.properties` and `application.yaml` actually are the same thing, but in different format only. Since we already use properties file, so just remove this yaml file.

In order to tell the DevOps that this property should be overiddent at runtime, add the following comment in application.properties:

`# kb.jwt.aws.userPoolId=`

Reply:
Generally, environment var is written in capital with underscore. So when the DevOps see this `kb.jwt.awt.userPoolId=` he understands that the variable name will be `USER_POOL_ID` ?

Reply:
yes, there is a naming convention for setting properties through env_var